### ISSUE TITLE
If someone want to update the token address in json file, how it can be done ?

### ISSUE THREAD

**Issue by solcatcoin** [solana-labs/token-list#4582]
If someone want to update the token address in json file, how it can be done ? As deletion is not allowed, once the commit is merged, how can we update the token contract address ?

**Comment by leoluk** [comment]
The PR needs to be reviewed manually

**Comment by Fiverr-Coin-Hunter** [comment]
> The PR needs to be reviewed manually

check this bro #4584

**Comment by Fiverr-Coin-Hunter** [comment]
> The PR needs to be reviewed manually

I will wait for more than one hour (still didn't merge) #4562

**Comment by solcatcoin** [comment]
> The PR needs to be reviewed manually

We got a situation at hand @leoluk, due to the merge being stopped for many days, we couldn't see which symbols have already been taken into SOLANA token list registry.
https://github.com/solana-labs/token-list/pull/4097 this PR raised by us, got in automerge-error tag today morning due to "MEW" token symbol already in token-list.
We have got over 17k+ members in discord & TWITTER and traded over $100k in 2 weeks time, token being traded on raydium and dexlab (https://trade.dexlab.space/#/market/9xTPHXTEnXC8eqmtDntDtKLsrq15Kbwy28ucaWJN2qzH), the other registry entry with "MEW" symbol is not active, without any website, twitter, or any latest transaction. Is there a way out for us, so that we can be manually verified. 

Thank you very much :)
#InUrgentNeedOfHelp

**Comment by solcatcoin** [comment]
https://github.com/solana-labs/token-list/pull/4576, this is the new PR raised by us, would be very very helpful for us if you can manually merge this up, we 17k CATS are awaiting on token symbol verification for long lime. Do hop in our discord cat family to see project's progress discord.gg/solcatcoin :)

**Comment by WhiteElephantEco** [comment]
dm me @asv1104 we can help for free. we just merged 2 tokens.
